### PR TITLE
Set go version to 1.19

### PR DESCRIPTION
**Pull request by revit13**
Signed-off-by: Revital Sur <eres@il.ibm.com>

### CODE REVIEW

**Review by cdoron — DISMISSED** [review]

File: Makefile [hunk 1]
```diff
@@ -97,5 +97,5 @@ test:
 check:
 	go fmt ./...
 	go vet ./...
-	go mod tidy -compat=1.17
-	docker run --rm -v ${PWD}:/app -w /app golangci/golangci-lint:v1.50.1 golangci-lint run
+	go mod tidy
+	docker run --rm -v ${PWD}:/app -w /app golangci/golangci-lint:v1.50.0 golangci-lint run
```
Comment (roytman) [inline Makefile:101]: why did you change `golangci-lint` to a previous version?

File: go.mod [hunk 2]
```diff
@@ -1,6 +1,6 @@
 module fybrik.io/openmetadata-connector
 
-go 1.17
+go 1.19
 
 require (
 	fybrik.io/fybrik v1.1.0
```
Comment (roytman) [inline go.mod:6]: should it be v1.2.0 ?

Comment (roytman) [reply go.mod:6]: you changed the go.mod, but I don't see changes in go.sum, did you run `go mod tidy` ?

Comment (revit13) [reply go.mod:6]: I ran `make check` but there were no changes. now there are changes because I upgraded Fybrik version.

File: Makefile (hunk 1 above)
Comment (revit13) [reply Makefile:101]: It a mistake after rebase. Fixed. Thanks

**Review by roytman — APPROVED** [review]
